src/fragments/lib/auth/ios/social_signin_web_ui/10_cli_setup.mdx

```diff
@@ -31,7 +31,7 @@ Choose the following options (the last steps are specific to Facebook here but a
     `<your Team ID>`
 ? Enter your Key ID for your OAuth flow:
     `<your Key ID>`
-? Enter your Private Key for your OAuth flow:
-    `<the content between -----BEGIN PRIVATE KEY----- and -----END PRIVATE KEY----- from your .p8 file in a single line>`
+? Enter your Private Key for your OAuth flow (entire key without line breaks):
+    `<the entire content -----BEGIN PRIVATE KEY----- <key>> -----END PRIVATE KEY----- from your .p8 file in a single line>`
```

Comment:
The previous note here was correct, and it was found that copying the entire key contents including the header and footer are causing https://github.com/aws-amplify/amplify-cli/issues/11526

Reply:
The corresponding changes are here https://github.com/aws-amplify/amplify-cli/pull/11790